VulkanHppGenerator.hpp

```diff
@@ -1054,6 +1058,7 @@ class VulkanHppGenerator
   std::string              toString( TypeCategory category );
 
 private:
+  DefinesPartition                        m_definesPartition;  // the partition of the defines into groups of mutually exclusive defines
```

Comment:
Being picky again: it's not "groups of mutually exclusive defines", but "mutually exclusive groups of defines".
Also, please move it down a little, right after the m_defines.